### DISCUSSION TITLE
FileSystem "writeback error on sector"

### DISCUSSION THREAD

**Discussion by squares-ui** [Security-Onion-Solutions/securityonion#2673] [Unsupported Versions] [answered]
Hi, 
Probably not an issue with SO2, but thought I would mention just in case it's of interest

Appears file system error caused a shutdown of services.  This is 2.3.10.iso running in ESXi 7.0.0 (Build 15843807)

![image](https://user-images.githubusercontent.com/21036586/105060475-5d1dc080-5a70-11eb-9f3f-92009224f8fc.png)
![image](https://user-images.githubusercontent.com/21036586/105060564-758ddb00-5a70-11eb-8573-10caa774ed2b.png)

Regards

**Comment by TOoSmOotH** [comment, marked as answer]
You are correct looks like you have an issue with your disk. This is not a SO2 issue.
